src/test/java/core/basesyntax/service/impl/FruitTransactionParserImplTest.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitTransactionParser;
+import java.util.List;
+import org.junit.Test;
+
+public class FruitTransactionParserImplTest {
```

Comment:
let's also add test where we should parse list with header only
and test with many transactions (more than 2 will be enough)